glide.lock

```diff
@@ -229,7 +226,7 @@ imports:
 - name: github.com/feyeleanor/slices
   version: bb44bb2e4817fe71ba7082d351fd582e7d40e3ea
 - name: github.com/fsouza/go-dockerclient
-  version: 76fd6c68cf24c48ee6a2b25def997182a29f940e
+  version: ""
```

Comment:
@bacongobbler after running `glide up` again (more precisely, `make glideup`), this version number didn't come back. In https://github.com/arschles/deis-minio/commit/08099c9503ea2908b00df85b422fd4eeee27dab8, however, another version number did come back for a launchpad.net dependency...

@technosophos suggested that I file an issue on glide for this, so I'm working on getting a solid set of repro steps down before I do.